### ISSUE TITLE
45th Alaska Anthropological Association annual meeting

### ISSUE THREAD

**Issue by vlraymond** [NCEAS/arctic-data-outreach#26]
March 21 - 24, 2018
Anchorage, AK

- [x] MJ / AB give greenlight for conference
- [x] VR to get talk moved to a more appropriate session
- [x] VR register, flights, hotel
- [x] VR prep slide deck using reading listed below and guidance from MJ / AB
- [x] VR get feedback from MJ / AB on deck.

[draft slide deck](https://docs.google.com/presentation/d/1BwTl5oMSmHwbtCL4__oLPxyC9PKDmeaCymWoN6lg9U4/edit#slide=id.g3128cddd84_0_33)

**Comment by vlraymond** [comment]
reading: https://www.sesync.org/system/tdf/resources/qualitative_data_sharing_and_re-use_white_paper.pdf?file=1&type=node&id=2536&force

approach:
offering a service
data resources
available forward of re-use of information
start a dialog, how to be helpful

consultation from CSH

**Comment by vlraymond** [comment]
The event organizers had put us in a session on "New approaches to data.. in Archaeology'. I'm working with them on finding a better fit. They're being amazingly receptive.  I have proposed the following as an alternative

> "New Approaches to Collaborative Exhibit Development..." session
> I could tailor my talk to focus on how data preservation and creation of quality metadata can support exhibition work. We have a "data provenance" tool and also some "data rescue" initiatives that could be good support for collaborative exhibits.

**Comment by aebudden** [comment]
I started to review the document. I can make some design comments however it's hard to comment on the flow without knowing the accompanying 'story'. Can we set up a call for you to talk through this presentation / story?

**Comment by vlraymond** [comment]
Sounds good Amber, whenever works.

**Comment by aebudden** [comment]
Can you identify times next week (early in your day bc I will be in Europe) that work for you and Matt. Once I have a better idea of my schedule I can select one and set up a meeting.

**Comment by vlraymond** [comment]
Monda 3/19
8am Pacific
9am Pacific
10am Pacific (MJ has a hold for NCO team meeting)
11am Pacific

Tue 3/20
8am Pacific (MJ has a hold for MDC)

Wed 3/21
8am Pacific
9am Pacific (MJ has a hold for MDC)

**Comment by aebudden** [comment]
OK, I have an evening thing on Monday so 8am is the best option. or Wednesday at 8am - I don't yet have my schedule for that day so it might be more risky.

**Comment by aebudden** [comment]
Let's use: https://zoom.us/my/aebudden. I send out a calendar invite to you, Matt and Kathryn. Feel free to invite others.

**Comment by vlraymond** [comment]
Deck has been updated to incorporate feedback from yesterday :>

**Comment by vlraymond** [comment]
Trip report
VR attended on 3/22/18 and 3/23/18. There were about 100 - 125 (not good at guessing numbers) attendees, from AK (UAA, UAS, UAF) and lower 48 institutions. Attendees were a mix of anthropologists, archaeologists, linguists, museum and library staff, and remote sensing people. Institutions represented included: federal and state agency staff, private firms, non-profits, and academics. Academics included lead researchers in the field (Walkie Charles, Larry Kaplan etc) down to graduate student level and even a few recently graduated undergraduates.

VR's talk on the 23rd was attended by ~20 to 25 museum and library staff from Smithsonian, Anchorage Museum, academics from AK and lower 48 institutions (Oberlin), and smaller museums and libraries in AK. Also in attendance were artists, archaeologists, and private company staff from a 3D imagery / VR company. Talk was very well received and there were many questions afterwards about ADC, ADC submission process, holdings, Austin Post dataset, and our data science trainings (fellowship and 5-day training). I saw people taking pictures of our slides so... I guess we did good work in making a clear and compelling argument :>

Outside of the conference VR had sidebar talks with:
- Emily Russell, Producer AK Public Radio
- Nate Bauer, Director, University of Alaska Press
- Anastasia Tarmann, Director, Digital Newspaper Project, AK State Historical Collections
